### PR TITLE
Added color support and changing code

### PR DESCRIPTION
**Pull request by Trion129**
Added "color" input type support in crawler to solve #81 

First the crawler clicks on the input, this puts it in view and then uses the included JS code to change the input value to red.

Verified on http://www.htmlquick.com/reference/tags/input-color.html

## Firefox screenshot:
image

## Chrome Screenshot
image

### CODE REVIEW

File: collect.py [hunk 1]
```diff
@@ -164,6 +164,9 @@ def do_something(driver, elem_attributes=None):
         elif input_type == 'search':
             elem.clear()
             elem.send_keys('quick search')
+        elif input_type == 'color':
+            elem.click()
```
Comment (marco-c) [inline collect.py]: Clicking on the element shouldn't be needed.

File: collect.py [hunk 2]
```diff
@@ -164,6 +164,9 @@ def do_something(driver, elem_attributes=None):
...
+            driver.execute_script("arguments[0].setAttribute('value', '#FF0000')", elem)
```
Comment (marco-c) [inline collect.py]: You can use `arguments[0].value = '#FF0000'`.
Or even better, use the Selenium API to do it

Comment (Trion129) [reply collect.py]: I tried to find a set_attribute function in Selenium WebDriver and WebElement API but it wasn't present. So I went for javascript's setAttribute.
The `.value` notation looks cleaner to me, will try it :)
I'll replace the `.click()` with an ActionChain, so I move to the element and set the value.

Comment (marco-c) [reply collect.py]: If you use `.value`, you don't need to do anything else before

**Comment by Trion129** [comment]
Tried various approaches:

First approach:
1. click()
2. setAttribute using executescript
Problem: There was inconsistency in the screenshot on the position of input, might have caused problem in model training. Also click isn't a very clean procedure for this purpose.

Second approach:
1. Directly setting `.value`
Problem: Unfortunately, Doesn't scroll the element in view

Third approach:
1. Use ActionChain to move to the element
2. Setting `.value`
Problem: For the example website, it errored out giving out of viewport error, weird!

Fourth approach:
1. Used JS to call `.scrollIntoView()`
2. Set the `.value`
Screenshots are consistent and show the color input as expected! More beneficial for model to learn from.

## Chrome:
image

## Firefox:
image

**Comment by marco-c** [comment]
The scrollIntoView is a good idea! It should be implemented in a separate PR though, and for all kinds of elements (not just color).

File: collect.py [hunk 3]
```diff
@@ -164,6 +164,9 @@ def do_something(driver, elem_attributes=None):
         elif input_type == 'search':
             elem.clear()
             elem.send_keys('quick search')
+        elif input_type == 'color':
+            driver.execute_script("arguments[0].scrollIntoView();", elem)
```
Comment (marco-c) [inline collect.py]: Let's remove this, we will add it in a separate PR.

**Pull request merged**